src/common/testHelpers/mockGraphQLResponse.ts

```diff
@@ -0,0 +1,26 @@
+import nock from 'nock';
+
+// Enable debug for nock by setting the debug variable
+// i.e., DEBUG=nock.* yarn test
+
+const mockGraphQLResponse = (
+  url: string,
+  assertRequestHeaders?: Record<string, string | undefined>,
+  response: Object = {},
+  responseCode: number = 200,
+) => {
+  return nock(url)
```

Comment:
This is super basic and intended to reduce copy/paste between graphql hook tests. We may want to extend it slightly but more complex cases should just use nock directly (assuming we agree to take this approach).

Reply:
I can dig how nock gets us to _fully_ test the lib at hand, instead of mocking things in such a way that you can miss some behavior that's worth testing.  I've seen nock usage get hairy & hard to maintain, and I've seen requests accidentally not mocked and they leak out from unit tests.  As long as we watch those 2 things really closely, I think this should work out great.

Reply:
If we make the baseURL unique per test we should hopefully reduce any collisions. Definitely something to keep in mind.